features/bevragen/protocollering.feature

```diff
@@ -0,0 +1,85 @@
+#language: nl
+
+Functionaliteit: Als burger wil ik zien wie welke gegegevens van mij heeft gezien of gebruikt
+  Zoeken en raadplegen van gegevens van burgers worden "geprotocolleerd" (formeel gelogd).
+
+  Protocollering moet gebeuren op twee niveau's:
+  - bij de RvIG op het niveau van afnemende organisaties
+  - bij de afnemende organisatie op het niveau van gebruikers
+
+  Deze feature beschrijft alleen de werking van protocollering op het niveau van afnemende organisaties bij de bron (RvIG)
+
+  Daarin worden de volgende gegevens vastgelegd:
+  - request_id: unieke identificatie van de berichtuitwisseling waarmee het protocolleringsrecord te koppelen is aan andere logging van de API
+  - request_datum: tijdstip waarop de protocollering is vastgelegd
+  - afnemer_code: identificatiecode van de afnemende organisatie, zoals die is opgenomen in de gestuurde Oauth 2 token
+  - anummer: het A-nummer van de persoon waarvan gegevens zijn gevraagd of geleverd
+  - request_zoek_rubrieken: spatie gescheiden lijst van alle gegevens die met fieds gevraagd zijn, vertaald naar LO-BRP elementnummers
+  - request_geleverde_rubrieken: spatie gescheiden lijst van alle feitelijk geleverde gegevens, vertaald naar LO-BRP elementnummers
+  - verwerkt: vaste waarde false (boolean)
+
+  Rule: Gevraagde velden in fields worden vertaald naar elementnummers volgens Logisch ontwerp BRP
+    Dit is een 6-cijferige code, met zo nodig voorloopnul voor categorieën. Bijvoorbeeld 010120 is het burgerservicenummer (categorie 01, groep 01 en rubriek 20).
+
+    Abstract Scenario: Gevraagde veld <fields veld> wordt vastgelegd als zoekrubriek <rubrieknummer>
+
+    Scenario: Meerdere velden gevraagd
+
+  Rule: Wanneer met fields een hele groep wordt gevraagd, worden alle velden in die groep vastgelegd bij zoekrubrieken
+
+    Scenario: vragen om hele groep <groep>
+
+    Scenario: vragen om hele relatie <relatie>
+
+  Rule: Wanneer een gevraagd veld niet geleverd wordt, wordt het wel bij zoekrubrieken en niet bij geleverde rubrieken opgenomen
```

Comment:
`Rule: Wanneer een gevraagd veld niet geleverd wordt, wordt het wel in het veld 'request_zoek_rubrieken' en niet in het veld 'request_geleverde_rubrieken' vastgelegd.`